server/tests/backends/fixtures/convert/from_datetime_to_text_pypika.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake",
+    "athena_pypika",
+    "bigquery_pypika"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "convert",
+        "columns": [
+          "brewing_date"
+        ],
+        "data_type": "text"
+      },
+      {
+        "name": "sort",
+        "columns": [
+          {
+            "column": "name",
+            "order": "asc"
+          }
+        ]
+      }
+    ]
```

Comment:
ni: could you please add a `select` step to have more readable fixtures ?

Reply:
y